docs/bluemix-postgres.md

```diff
@@ -1,57 +1,65 @@
 
 
-1. [Install Docker CLI and Bluemix Container registry Plugin](#1-install-docker-cli-and-bluemix-container-registry-plugin)
-2. [Create Compose for PostgreSQL on Bluemix](#2-create-compose-for-postgresql-on-bluemix)
+1. [Install Docker CLI and IBM Cloud Container registry Plugin](#1-install-docker-cli-and-ibm-cloud-container-registry-plugin)
+2. [Create Databases for PostgreSQL on IBM Cloud](#2-create-databases-for-postgresql-on-ibm-cloud)
 3. [Build Gitlab container](#3-build-gitlab-container)
 4. [Create Services and Deployments](#4-create-services-and-deployments)
 5. [Using Gitlab](#5-using-gitlab)
 
-# 1. Install Docker CLI and Bluemix Container Registry Plugin
+# 1. Install Docker CLI and IBM Cloud Container Registry Plugin
 
 
 First, install [Docker CLI](https://www.docker.com/community-edition#/download).
 
-Then, install the Bluemix container registry plugin.
+Then, install the IBM Cloud container registry plugin.
 
 ```bash
-bx plugin install container-registry -r bluemix
+ibmcloud plugin install container-registry
 ```
 
-Once the plugin is installed you can log into the Bluemix Container Registry.
+Once the plugin is installed you can log into the IBM Cloud Container Registry.
 
 ```bash
-bx cr login
+ibmcloud cr login
 ```
 
-If this is the first time using the Bluemix Container Registry you must set a namespace which identifies your private Bluemix images registry. It can be between 4 and 30 characters.
+If this is the first time using the IBM Cloud Container Registry you must set a namespace which identifies your private IBM Cloud images registry. It can be between 4 and 30 characters.
 
 ```bash
-bx cr namespace-add <namespace>
+ibmcloud cr namespace-add <namespace>
 ```
 
 Verify that it works.
 
 ```bash
-bx cr images
+ibmcloud cr images
 ```
 
-# 2. Create Compose for PostgreSQL on Bluemix
+# 2. Create Databases for PostgreSQL on IBM Cloud
 
-Use the Bluemix catalog or the `bx` command to create a service instance of Compose for PostgreSQL and add a set of credentials.
+Use the IBM Cloud catalog or the `ibmcloud` command to create a service instance of `databases-for-postgresql-gl` and add a set of credentials.
 
 ```bash
-bx service create compose-for-postgresql Standard "Compose for PostgreSQL-GL"
-bx service key-create "Compose for PostgreSQL-GL" Credentials-1
+ibmcloud resource service-instance-create databases-for-postgresql-gl databases-for-postgresql standard <SERVICE_PLAN_NAME LOCATION>
+For example, in `us-south`: ibmcloud resource service-instance-create databases-for-postgresql-gl databases-for-postgresql standard us-south
+
+ibmcloud resource service-key-create --instance-name databases-for-postgresql-gl Credentials-1 Administrator
+```
+
+Set the PostgreSQL database aministrator password.
+
+```bash
+ibmcloud cdb user-password hick-databases-for-postgresql-gl admin <admin_password>
```

Comment:
is this a dig at the datacenter in us-south?  we're all hicks, or something?  ;)

Reply:
:D I kinda like the idea of hick-databases....